### ISSUE TITLE
Trainable Parameters: 0.000M

### ISSUE THREAD

**Issue by zhiyongLiu1114** [babysor/MockingBird#723]
**Summary[问题简述（一句话）]**

您好，请问我在训练synthesizer 的时候，显示Trainable Parameters: 0.000M是正常的吗？看起来是没有参数在训练

**Env & To Reproduce[复现与环境]**
描述你用的环境、代码版本、模型

**Screenshots[截图（如有）]**
If applicable, add screenshots to help

**Comment by babysor** [comment]
不正常，有更多截图或信息吗

**Comment by CatOverrun** [comment]
> 不正常，有更多截图或信息吗

遇到同样的现象

`PS I:\wav\list\MockingBird> python .\synthesizer_train.py mandarin I:\wav\list\Dataset\SV2TTS\synthesizer
Arguments:
    run_id:          mandarin
    syn_dir:         I:\wav\list\Dataset\SV2TTS\synthesizer
    models_dir:      synthesizer/saved_models/
    save_every:      1000
    backup_every:    25000
    log_every:       200
    force_restart:   False
    hparams:

Checkpoint path: synthesizer\saved_models\mandarin\mandarin.pt
Loading training data from: I:\wav\list\Dataset\SV2TTS\synthesizer\train.txt
Using model: Tacotron
Using device: cuda

Initialising Tacotron Model...

\Loading the json with %s
 {'sample_rate': 16000, 'n_fft': 800, 'num_mels': 80, 'hop_size': 200, 'win_size': 800, 'fmin': 55, 'min_level_db': -100, 'ref_level_db': 20, 'max_abs_value': 4.0, 'preemphasis': 0.97, 'preemphasize': True, 'tts_embed_dims': 512, 'tts_encoder_dims': 256, 'tts_decoder_dims': 128, 'tts_postnet_dims': 512, 'tts_encoder_K': 5, 'tts_lstm_dims': 1024, 'tts_postnet_K': 5, 'tts_num_highways': 4, 'tts_dropout': 0.5, 'tts_cleaner_names': ['basic_cleaners'], 'tts_stop_threshold': -3.4, 'tts_schedule': [[2, 0.001, 10000, 12], [2, 0.0005, 15000, 12], [2, 0.0002, 20000, 12], [2, 0.0001, 30000, 12], [2, 5e-05, 40000, 12], [2, 1e-05, 60000, 12], [2, 5e-06, 160000, 12], [2, 3e-06, 320000, 12], [2, 1e-06, 640000, 12]], 'tts_clip_grad_norm': 1.0, 'tts_eval_interval': 500, 'tts_eval_num_samples': 1, 'tts_finetune_layers': [], 'max_mel_frames': 900, 'rescale': True, 'rescaling_max': 0.9, 'synthesis_batch_size': 16, 'signal_normalization': True, 'power': 1.5, 'griffin_lim_iters': 60, 'fmax': 7600, 'allow_clipping_in_normalization': True, 'clip_mels_length': True, 'use_lws': False, 'symmetric_mels': True, 'trim_silence': True, 'speaker_embedding_size': 256, 'silence_min_duration_split': 0.4, 'utterance_min_duration': 1.6, 'use_gst': True, 'use_ser_for_gst': True}
Trainable Parameters: 0.000M

Loading weights at synthesizer\saved_models\mandarin\mandarin.pt
Tacotron weights loaded from step 79000
Using inputs from:
        I:\wav\list\Dataset\SV2TTS\synthesizer\train.txt
        I:\wav\list\Dataset\SV2TTS\synthesizer\mels
        I:\wav\list\Dataset\SV2TTS\synthesizer\embeds
Found 439 samples
+----------------+------------+---------------+------------------+
| Steps with r=2 | Batch Size | Learning Rate | Outputs/Step (r) |
+----------------+------------+---------------+------------------+
|   81k Steps    |     20     |     5e-06     |        2         |
+----------------+------------+---------------+------------------+

{| Epoch: 1/3682 (22/22) | Loss: 0.1255 | 1.1 steps/s | Step: 79k | }}
{| Epoch: 2/3682 (22/22) | Loss: 0.1242 | 1.2 steps/s | Step: 79k | }
{| Epoch: 3/3682 (6/22) | Loss: 0.1219 | 1.2 steps/s | Step: 79k | }

`

**Comment by GreenDragonQueen** [comment]
遇到了同样的问题：
D:\MockingBird-main>python synthesizer_train.py kilruk D:\BaiduNetdiskDownload\SV2TTS\synthesizer
Arguments:
    run_id:          kilruk
    syn_dir:         D:\BaiduNetdiskDownload\SV2TTS\synthesizer
    models_dir:      synthesizer/saved_models/
    save_every:      1000
    backup_every:    25000
    log_every:       200
    force_restart:   False
    hparams:

Checkpoint path: synthesizer\saved_models\kilruk\kilruk.pt
Loading training data from: D:\BaiduNetdiskDownload\SV2TTS\synthesizer\train.txt
Using model: Tacotron
Using device: cuda

Initialising Tacotron Model...

\Saving the json with %s
 synthesizer\saved_models\kilruk\kilruk.json
Trainable Parameters: 0.000M

Loading weights at synthesizer\saved_models\kilruk\kilruk.pt
Tacotron weights loaded from step 0
Using inputs from:
        D:\BaiduNetdiskDownload\SV2TTS\synthesizer\train.txt
        D:\BaiduNetdiskDownload\SV2TTS\synthesizer\mels
        D:\BaiduNetdiskDownload\SV2TTS\synthesizer\embeds
Found 63 samples

**Comment by BingQZh** [comment]
我也有一样的问题，但synthesizer训练出来又好像是有效的……

**Comment by zhiyongLiu1114** [comment]
> 

和后面那些人的评论一样的
